### PR TITLE
ALBS-668: PES: Allow to set different OS names for 7, 8, 9 versions

### PR DESCRIPTION
**Pull request by soksanichenko**
- The subject is implemented

### CODE REVIEW

File: src/backend/api/handlers.py [hunk 1]
```diff
@@ -180,12 +180,12 @@ def modify_action(action_data: ActionData) -> None:
 
 
 def dump_pes_json(
-        source_release: str,
-        target_release: str,
+        oses: dict[int, str],
```
Comment (javihernandez) [inline src/backend/api/handlers.py:183]: Just to double check, is `oses` coming the following way?
```
{
  8: 'AlmaLinux 8',
  9: 'AlmaLinux 9'
}
```

If so, why forcing the key to be int in line 188?

Comment (soksanichenko) [reply src/backend/api/handlers.py:183]: No, it comes as 

```
{
  8: 'AlmaLinux',
  9: 'AlmaLinux',
  7: 'CentOS',
}
```

I mean a name of OS doesn't contain major version.

File: src/backend/templates/bulk_upload.html [hunk 2]
```diff
@@ -25,9 +22,9 @@
                 </div>
                 <div class="row">
                     <div class="col-sm">
-                        {{ form.org.label }}
-                    </div>
-                    <div class="col-sm">
+                        {{ form.org.label }}<br><br>
+{#                    </div>#}
```
Comment (javihernandez) [inline src/backend/templates/bulk_upload.html]: Is there a good reason to keep this commented? Can't they just go away?

File: src/backend/templates/dump.html [hunk 3]
```diff
@@ -5,79 +5,63 @@
...
+                    </div>
+                    <div class="col">
+                        &nbsp;
+                    </div>
+                </div>
+                &nbsp;
+                <div class="row">
+                    <div class="col">
+                        {{ form.groups.label }}:
+{#                    </div>#}
```
Comment (javihernandez) [inline src/backend/templates/dump.html]: Same as previous comment

**Review by javihernandez — COMMENTED** [review]
Left a few minor comments but, to the extent I can tell, it looks good to me.

File: src/backend/templates/dump.html (hunk 3 above)
Comment (soksanichenko) [reply src/backend/templates/dump.html]: Done

File: src/backend/templates/bulk_upload.html (hunk 2 above)
Comment (soksanichenko) [reply src/backend/templates/bulk_upload.html]: Done

**Review by javihernandez — APPROVED** [review]

**Pull request merged**